### PR TITLE
Fixed issue #17021: Saving question fails

### PR DESCRIPTION
**Pull request by gabrieljenik**

### CODE REVIEW

File: application/controllers/QuestionAdministrationController.php [hunk 1]
```diff
@@ -2482,14 +2482,12 @@ private function applyL10n($oQuestion, $dataSet)
...
-                    'help'     => $aI10NBlock['help'],
-                    'script'   => $aI10NBlock['script'],
-                ],
-                false
-            );
+            $aAttributes = [
+                'question' => $aI10NBlock['question'],
+                'help'     => $aI10NBlock['help'],
+            ];
+            if (array_key_exists('script', $aI10NBlock)) $aAttributes['script'] = $aI10NBlock['script'];    // Script is not present when "Show script field" = "No"
```
Comment (Shnoulle) [inline application/controllers/QuestionAdministrationController.php:2489]: PSR12 please :)

Comment (gabrieljenik) [reply application/controllers/QuestionAdministrationController.php]: Done. Thanks!

**Comment by olleharstedt** [comment]
I accidentally made another fix. Any difference?

**Comment by gabrieljenik** [comment]
> I accidentally made another fix. Any difference?

Which one?

**Comment by olleharstedt** [comment]
See Denis' comment.

**Comment by gabrieljenik** [comment]
I think this line is not in that fix

**Comment by olleharstedt** [comment]
It is, but inlined.

**Comment by olleharstedt** [comment]
Or rather, setAttributes() was not changed in my commit.

**Comment by gabrieljenik** [comment]
You are right.

**Comment by olleharstedt** [comment]
Guess I'll close this one then? Sorry to steal your thunder. :|

**Comment by gabrieljenik** [comment]
No worries

**Pull request closed without merging**